### ISSUE TITLE
push hcpy to hub.docker.com

### ISSUE THREAD

**Issue by nicx** [hcpy2-0/hcpy#114]
Hi, could you just push the current state to the registry hub.docker.com? As far as I can see the Github Actions workflow is already in place. What's missing for the push to docker hub? Would be great, even in the beta state :)

**Comment by Hypfer** [comment]
It is generally considered bad practice to just repeat a request when you don't get an answer

Here's you asking the same thing:
https://github.com/hcpy2-0/hcpy/discussions/112

I don't believe that anyone is willing to shoulder the extra support burden for what you're requesting, though that is just my personal guess.

**Comment by nicx** [comment]
@Hypfer you are right, sorry for that. I just didn't receive any response, that's wha I thought the "discussions" are maybe the wrong place?! 

Before I fork the project just for that reason I would try to support this myself, even if I'm not a developer. Is that in the interests of the project maintainers?

**Comment by Hypfer** [comment]
Given the overall very prototype state of the project, I'd guess that there isn't any interest in that yet.
There may be in the future and then there perhaps depending on maintainer decisions be official docker images

For now however it's probably just you solving your own problem on your own

**Comment by Meatballs1** [comment]
I think it just needs a release creating for the docker image, I've created a new one and the CI is whirring along... Might be another step to complete to publish the HA addon. Will see if I can remember how or if I need @pmagyar to approve it

**Comment by nicx** [comment]
Ill got it running with using ghcr.io/hcpy2-0/hcpy as repo. so never mind, I think its ok this way as long as this project is in prototype state. thanks for your help :)

**Comment by pmagyar** [comment]
Hello,
we use the github registry and upload the images there.
This has the advantage that we have everything within the github project and don't have to use external services/share access.
This means that everyone can continue to run the project.
Yes, you can pull the images with "ghcr.io/hcpy2-0/hcpy".
We will expand the documentation and write a guide for it.
